configure

```diff
@@ -435,6 +435,7 @@ check_incdir() {
       "$incdir/lua$LUA_VERSION/lua.h" \
       "$incdir/lua$(echo "$LUA_VERSION" | tr -d .)/lua.h" \
       "$incdir/lua.h" \
+      "$incdir/lua-$LUA_VERSION/lua.h" \
```

Comment:
I think lua.h is meant to be checked last. Could you move this up a line or two?